ftw/linkchecker/configuration.py

```diff
@@ -0,0 +1,51 @@
+from Products.CMFDiffTool.utils import safe_utf8
+import argparse
+import json
+import logging
+import os
+
+
+class Configuration(object):
+
+    def __init__(self, args):
+        self.config_file_path = ''
+        self.log_file_path = ''
+        self.sites = []
+
+        self._parse_arguments(args)
+        self._validate_args()
+        self._extract_config()
+
+    def _parse_arguments(self, args):
+        parser = argparse.ArgumentParser()
+        parser.add_argument("--config",
+                            help="Path to linkchecker config file.")
+        parser.add_argument("--log",
+                            help="Path to linkchecker log file.")
+        args, unknown = parser.parse_known_args()
```

Comment:
I'm not sure if you should use `parse_known_args`. Typing errors are not recognized this way. Are the arguments additionally parsed elsewhere?

Reply:
I created an issue to look at that. It is not really part of the PR since it was only copy pasted to a new location.